src/amo/components/Home/index.js

```diff
@@ -43,6 +45,33 @@ export const isFeaturedCollection = (
   });
 };
 
+export const getFeaturedCollectionsMetadata = (i18n: I18nType) => {
+  return [
+    {
+      footerText: i18n.gettext(
+        'See more social media customization extensions',
+      ),
+      header: i18n.gettext('Social media customization'),
+      ...FEATURED_COLLECTIONS[0],
```

Comment:
Ugh, it's awkward that the metadata will be defined in two places. I don't have a better idea though.

Reply:
Yeah, the need for `i18n` for the header and footer text leaves me in this position. :-(